x-pack/plugins/code/server/security.ts

```diff
@@ -43,6 +43,7 @@ export class SecureRoute {
                 const userRoles = new Set(req.auth.credentials.roles || []);
                 const authorized =
                   userRoles.has('superuser') ||
+                  userRoles.has('code_admin') ||
```

Comment:
@spacedragon do we need to remove the `superuser` check in here?

Reply:
I'd rather not. The default user like `elastic` has full access to elasticsearch should also regard as code_admin, I think

Reply:
Btw, why do we need add `userRoles.has('code_admin')` here? 
``` requiredRoles = requiredRoles.concat([ADMIN_ROLE]);```

Reply:
OK, now this makes sense. The 'code_admin' has been checked already. so yes, this change is not necessary.